docs/components/networktransform.md

```diff
@@ -51,8 +51,14 @@ darkImageSrc="/img/BufferedTick_Dark.png?text=DarkMode"/>
 
 `NetworkTransform` always synchronizes positions from the server to the clients and position changes on the clients are not allowed. Netcode for GameObjects comes with a sample containing a `ClientNetworkTransform`. This transform synchronizes the position of the owner client to the server and all other client allowing for client authoritative gameplay.
 
-The `ClientNetworkTransform` lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`
 
-Or you can directly add this line to your `manifest.json` file:
+You can use the existing `ClientNetworkTransform` that lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`.
```

Comment:
Is Git required to be installed prior to attempting to add the package via from Git URL?

Reply:
[It doesn't say you need Git installed ](https://docs.unity3d.com/Manual/upm-ui-giturl.html) in order to install a package from a Git URL. This might "just work" with the package manager.

Reply:
Could we reword this a little bit:
> You can use the existing `ClientNetworkTransform` in the Multiplayer Samples Utilities package. 
> To add the Multiplayer Samples Utilities package:
> 1. Open the Package Manager by selecting **Window** > **Package Manager**.
> 2. Select the **plus button (+)** > **Add from git URL...**.
> 3. Copy and paste the following Git URL: `https://github.com/Unity-> Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`
> 4. Select **Add**.

Reply:
@armstrongl 
Done!